corehq/form_processor/utils/general.py

```diff
@@ -26,7 +26,18 @@ def clear_local_domain_sql_backend_override(domain):
     _thread_local.use_sql_backend = use_sql_backend_dict
 
 
+def _short_circuit_sql_check(domain_object_or_name):
+    from corehq.apps.domain.models import Domain
+    domain_name = domain_object_or_name
+    if isinstance(domain_object_or_name, Domain):
+        domain_name = domain_object_or_name.name
+
+    return ENTERPRISE_OPTIMIZATIONS.enabled(domain_name)
```

Comment:
would be great to set this up for the `enikshay` domain too, although I'm not certain what the consequences of using this feature flag are on softlayer (the `enikshay` domain is on SQL, and we know that for a fact, so maybe it will be inconsequential).